backend/src/errors/service.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * This file contains errors pertaining to errors in service
+ * functions
+ */
+
+import { CustomError } from "./errors";
+
+const IMAGE_NOT_SAVED =
+  "Image was not able to be saved. Be sure to specify that image key is image";
+const IMAGE_NOT_FOUND = "Image was not found. Please make sure id passed in route is valid";
+const INVALID_MONGO_ID = "Mongo Id was invalid. Please ensure that the id is correct";
+const MENTEE_WAS_NOT_FOUND = "Mentee was not found";
+const MENTOR_WAS_NOT_FOUND = "Menor was not found";
+const IMAGE_WAS_NOT_FOUND = "Image was not found";
+
+export class ServiceError extends CustomError {
+  static IMAGE_NOT_SAVED = new ServiceError(0, 402, IMAGE_NOT_SAVED);
```

Comment:
HTTP 402 code means "Payment Required" which does not go well here. 

You could use [404 - Not Found](http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.4.5) for these cases